cli/tsc/00_typescript.js

```diff
@@ -47332,7 +47332,7 @@ var ts;
         }
         function bindSpecialPropertyAssignment(node) {
             var _a;
-            // Class declarations in Typescript do not allow property declarations
+            // Class declarations in TypeScript do not allow property declarations
```

Comment:
This is what exists in the upstream TypeScript projects and is copied directly from their distributions.  If you want to fix this, you will need to raise a PR at https://github.com/microsoft/TypeScript.